src/app/elements/kc-workspace-frame/kc-parts-controls.ts

```diff
@@ -239,6 +240,9 @@ export class KCPartsControls extends LitElement {
         if (existing) {
             return 'A part with that name already exist';
         }
+        if (!newName.length) {
+            return 'A part name can\'t be empty';
```

Comment:
could we add the i18n to this file?

Reply:
Yes! I forgot about that. I'll do that now.

Reply:
@rachaelcodes The validation errors should be translated now.